tests/foreman/api/test_syncplan.py

```diff
@@ -50,27 +50,24 @@
 from robottelo.utils.issue_handlers import is_open
 
 
-@filtered_datapoint
-def valid_sync_dates():
-    """Returns a list of valid sync dates."""
-    return [
-        # Today
-        datetime.now(),
-        # 5 minutes from now
-        datetime.now() + timedelta(seconds=300),
-        # 5 days from now
-        datetime.now() + timedelta(days=5),
-        # Yesterday
-        datetime.now() - timedelta(days=1),
-        # 5 minutes ago
-        datetime.now() - timedelta(seconds=300),
-    ]
+sync_date_deltas = {
```

Comment:
This is the core thing that I'm addressing in the PR - moving parametrization from datetime instances to a signed int used for timedelta kwargs within the test cases.